src/Core/Renderer/Breadcrumbs/BreadcrumbsHelper.php

```diff
@@ -115,6 +115,35 @@ private function getPrevPage(string $templateName): ?string
         return $prevPage;
     }
 
+    /**
+     * @throws InvalidConfigurationParameterException
+     */
+    public function getNearestIndexFile(string $templateName): string
+    {
+        $pathParts = explode('/', $templateName);
+        array_pop($pathParts);
+        $subPath = implode('/', $pathParts);
+        $finder = Finder::create()
```

Comment:
We've the same code-block on lines 94->108 of this file, might be worth extracting to its own method?

Reply:
fixed